multi_label_trainer_tfidf.ipynb

```diff
@@ -4,10 +4,9 @@
   "metadata": {
```

Comment:
<p>Let's add some sample inference post evaluation. If the results are not up to the mark, we can add suggestions as to how to get better results in the conclusion section.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/soumik12345/multi-label-text-classification/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='41'/>